utils/helper.sh

```diff
@@ -82,8 +75,8 @@ function check_python() {
 }
 
 function check_confluent_binary() {
-  if [[ ! -f ~/confluent-v1/confluent ]]; then
-    echo "The alias 'confluent-v1' is not set up. Follow the migration guide (https://docs.confluent.io/confluent-cli/current/migrate.html#run-multiple-clis-in-parallel) and try again."
+  if [[ ! -f ~/confluent/confluent ]]; then
+    echo "The alias 'confluent' is not set up. Use the Confluent CLI bundled with Confluent Platform and try again."
```

Comment:
We're no longer using an alias here, so let's revert to the original logic.
```suggestion
  if [[ $(type confluent 2>&1) =~ "not found" ]]; then
    echo "'confluent' is not found. Install Confluent Platform if you want to use Confluent CLI."
```

Reply:
Ack and done in latest commit